querybook/server/datasources/ai_assistant.py

```diff
@@ -5,9 +5,10 @@
 from lib.ai_assistant import ai_assistant
 
 
-@register("/ai/query_title/", methods=["POST"], custom_response=True)
+@register("/ai/query_title/", custom_response=True)
```

Comment:
is it possible to keep this as post?

Reply:
SSE doesn't support post method. if we'd like to keep post, we can switch back to use `fetch` and parse it accordingly.